pkg/logstash.sysv

```diff
@@ -141,7 +141,6 @@ configtest() {
     return 1
   fi
 
-  JAVA_OPTS=${LS_JAVA_OPTS}
   HOME=${LS_HOME}
   export PATH HOME JAVA_OPTS LS_HEAP_SIZE LS_JAVA_OPTS LS_USE_GC_LOGGING
```

Comment:
I see, so for my understanding https://github.com/elastic/logstash/blob/master/bin/logstash.lib.sh#L90 will do what you say, but only when using the vendored ruby, not when using ```USE_RUBY```, right? I might be missing something, but....

keeping works for me, we can always remove it later if possible.